dev-perl/Math-PlanePath/Math-PlanePath-123.0.0.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DIST_AUTHOR="KRYDE"
+DIST_VERSION="123"
+inherit perl-module
+
+DESCRIPTION="Mathematical paths through the 2-D plane"
+
+LICENSE="GPL-3"
```

Comment:
This should be GPL-3+ I think

from lib/Math/PlanePath.pm:
"either version 3, or (at your option) any later version."